public/app/features/scopes/internal/ScopesSelectorScene.tsx

```diff
@@ -207,6 +207,16 @@ export class ScopesSelectorScene extends SceneObjectBase<ScopesSelectorSceneStat
     this.setState({ isPickerOpened: false });
   }
 
+  public toggleDashboardsPanel() {
+    const dashboards = this.state.dashboards?.resolve();
+
+    if (this.state.scopes.length) {
```

Comment:
Just a super nit thing. I prefer explicit conditions.

```suggestion
    if (this.state.scopes.length > 0) {
```